src/libs/SearchUIUtils.ts

```diff
@@ -945,6 +946,38 @@ function createReportActionsByTransactionIDMap(data: OnyxTypes.SearchResults['da
     return reportActionsByTransactionID;
 }
 
+/**
+ * Calculates the "to" field value for a given transaction item based on the associated report and search data.
+ */
+function getToFieldValueForTransaction(
+    transactionItem: SearchTransaction,
```

Comment:
We're deprecating `SearchTransaction`, can we use src/types/onyx/Transaction` instead?